src/runtime/components/forms/Form.vue

```diff
@@ -29,7 +30,7 @@ export default defineComponent({
         | PropType<ZodSchema>
         | PropType<YupObjectSchema<any>>
         | PropType<JoiSchema>
-        | PropType<ValibotObjectSchema<any>>,
+        | PropType<ValibotSchema | ValibotSchemaAsync>,
```

Comment:
We need to extend the type definition for it to work with `v.safeParser`, something like this seems to work:

```ts
      type: [Object, Function] as
        | PropType<ZodSchema>
        | PropType<YupObjectSchema<any>>
        | PropType<JoiSchema>
        | PropType<ValibotSchema | ValibotSchemaAsync>
        | PropType<ValibotSafeParserAsync<ValibotSchemaAsync, any>>
        | PropType<ValibotSafeParser<ValibotSchema, any>>,
```

Reply:
Sorry, I forgot this part. Should I change it to:

```ts
PropType<ValibotSchema30 | ValibotSchemaAsync30 | ValibotSchema | ValibotSchemaAsync | ValibotSafeParser<any, any> | ValibotSafeParserAsync<any, any>>
```

or do you prefer to split it:

```ts
| PropType<ValibotSchema30 | ValibotSchemaAsync30>
| PropType<ValibotSchema | ValibotSchemaAsync>
| PropType<ValibotSafeParser<any, any> | ValibotSafeParserAsync<any, any>>
```

Reply:
Let's split it! I think we need to add the `Function` too, otherwise vue will throw a warning when using `:schema="v.safeParser(schema)"`

Reply:
Done